### ISSUE TITLE
[E::bwa_idx_load_from_disk] fail to locate the index files

### ISSUE THREAD

**Issue by xxYaaoo** [xunchen85/ERVcaller#21]
Hello, Dr. Chen~! I met a problem when I tried to run test data. The .err log showed me the detailed infor below:
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/25a52ed1-2d2d-4888-9806-450581a28465)
and here was my command line:
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/2be90080-0cf9-4156-bc53-9def81b4ceca)
how could I solve these problems? 
thank you for your help!

**Comment by xunchen85** [comment]
Hi,

If you correctly indexed the human genome and TE consensus sequences, the error from the very early bwa alignment step may be because no chimeric/split reads were identified. 

may I get a list of intermediate output files through "ls -lh" under your output folder? 

Best,
Xun

**Comment by xxYaaoo** [comment]
Thank you for your reply! 
My output folder:
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/a933378a-29cc-4791-9e98-9bab84bab5f4)
Besides, I am wondering....what you mean "correctly indexed the human genome and TE consensus sequences"....
Is there any step needed to be done about the human genome and TE consensus sequences before running the ERVcaller_v1.4.pl?

**Comment by xunchen85** [comment]
yes, when you prepared the reference files before running ERVcaller.pl, you need to run the "bwa index" command for both human and TEs. You could try it first and then let me know if you still have the same issue.

Best,
Xun

**Comment by xxYaaoo** [comment]
Hi, Dr. Chen! I think I forgot to "bwa index hg38.fa" previously. After running this command, I ran the ERVcaller command line again. 
This is my output folder: (while the .vcf file is still empty...). I feel there might still have some problems...
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/292f9b33-2d40-4370-8194-865cfe031a85)
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/8ec5d2d9-6a7e-43f4-8537-13c83d024fed)
Thank you for your help！

**Comment by xunchen85** [comment]
Hi, 

Have you also indexed your TE consensus sequences? Can you also share the log file and the file sizes under the temp folder?

Xun

**Comment by xxYaaoo** [comment]
Yes, I have checked my notes that I had indexed  the TE consensus sequences before.
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/204e103b-a010-4a40-ad84-8a0ae817f00d)
This is my temp folder:
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/ccb8c45f-6012-4dc7-8594-dbcb2ab8e321)
My slurm.out file:
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/51633e4f-2082-4060-b30f-55fa82e63a9c)
Head part of my slurm.err file: 
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/4967a454-f800-4bf6-a742-4b3c1d5e4e33)
Thank you so much!!!

**Comment by xunchen85** [comment]
I can't find any problem with your log and temp files.

Could you try using the BAM file or the TE_seq fastq file as the inputs? (not TE_seq2 which may not contain simulated insertions and used for testing separate FASTQ inputs) 

Best,
Xun

**Comment by xxYaaoo** [comment]
YEAH, Dr. Chen!~ I used the BAM file and it seemed like success?!
![image](https://github.com/xunchen85/ERVcaller/assets/149132410/c3a8cecd-c724-4daf-bd8e-d9fb100f006b)
And I am curious that the slurm.err file will not be empty, even if the running is successful?
(I will try to figure out the problems related with .fq.gz files and further run my own data.

**Comment by xunchen85** [comment]
Hi, 

I am glad that it works! 

I don't know what is included in your slurm.err file, but sometimes it is just the log BWA or samtools running which should be fine.

Sure, let me know if you have other questions. As I suggested, TE_seq files contacted the simulated integration sites but TE_seq2 may not, which could be the potential issue.

Best,
Xun

**Comment by xxYaaoo** [comment]
Dear Dr. Chen,

I have run my own data successfully using ERVcaller! You really make my these days! Thank you so much! Appreciate~

Best wishes,
Yaaoo
